src/fields/CheckBox.jsx

```diff
@@ -14,7 +14,7 @@ export default class CheckBox extends Component {
     return (
       <div>
         <label className={labelClass} ref="label">
-          <input {...cleanProps(props)} type="checkbox" ref="box"/>
+          <input {...cleanProps(props)} type="checkbox" ref="box" checked={!!props.value} />
```

Comment:
on an HTML note, this should be `null` or the string `checked`. For "normally assignment-less attributes" like this, the spec says to use the attribute name as its value when you use it, so `checked="true"` should be `checked="checked"`

Reply:
I think React will take care of it when checkedbox gets render - if `checked` is set to `false` the final `<input>` in DOM won't include a `checked` attribute in it